tech-test/src/components/SearchResults.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "../styles/SearchResults.css";
+
+const SearchResults = ({ results }) => {
+  if (!results.length) {
+    return <p>No results found</p>
+  } else { 
+    return (
+      <div className="search-results">
+        {results.map((image) => (
+          <img
+            className="card-image"
+            src={image}
+            alt="result-tile"
+            key={image}
+          />
+        ))}
+      </div>
+    )
+  };
```

Comment:
You dont need to add semi colons after the closing } braces, just after the closing ) brackets
```
    );
  }
```